doc/develop/test/twister.rst

```diff
@@ -410,13 +410,16 @@ harness: <string>
     harness implementation at the momemnt.
     The console harness tells Twister to parse a test's text output for a regex
     defined in the test's YAML file.
+    The Renode test harness is used to execute Robot Framework test suites
+    in the Renode simulation framework.
 
     Supported harnesses:
 
     - ztest
     - test
     - console
     - pytest
+    - renode_test
```

Comment:
can we put something generic here, like robot, that initially will be limited to renode but can be expanded to other simulators, and real hardware?

Reply:
renamed to `robot` as suggested